snapshot-labs/snapshot-sequencer
# fix: Allow `shutter` in proposal privacy if space is using `shutter` privacy

Discussion https://github.com/snapshot-labs/sx-monorepo/pull/954#discussion_r1867920957

### Summary

- Modifies privacy check to Allow `shutter` in proposal privacy if space is using `shutter` privacy

### How to test

- Use UI of https://github.com/snapshot-labs/sx-monorepo/pull/954
- Try changing privacy settings to `shutter`
- Try creating a proposal 
- Proposal should go in without any error (also with old UI)
